install/kubernetes/cilium/values.yaml

```diff
@@ -45,11 +45,11 @@ k8sServicePort: ""
 k8sClientRateLimit:
   # -- (int) The sustained request rate in requests per second.
   # @default -- 5 for k8s up to 1.26. 10 for k8s version 1.27+
-  qps:
+  qps: 10.0
   # -- (int) The burst request rate in requests per second.
   # The rate limiter will allow short bursts with a higher rate.
   # @default -- 10 for k8s up to 1.26. 20 for k8s version 1.27+
-  burst:
+  burst: 20
```

Comment:
I think this is being dynamically selected by the cilium-agent, so hardcoding a default in the Helm chart is working against the desired configuration (based on the local environment).